.github/actions/install-dashboards/action.yml

```diff
@@ -39,8 +39,8 @@ runs:
 
     - id: osd-version
       run: |
-        echo "::set-output name=osd-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-4)"
-        echo "::set-output name=osd-x-version::$(cat package.json | jq '.opensearchDashboards.version' | cut -c 2-3)"
+        echo "::set-output name=osd-version::$(jq -r '.opensearchDashboards.version' package.json)"
+        echo "::set-output name=osd-x-version::$(jq -r '.opensearchDashboards.version | split(".") | .[:2] | join(".")' package.json)"
```

Comment:
Here's what's happening in the second command:

- split("."): This splits the version string into an array using the period as a delimiter.
- .[:2]: This gets the first two elements of the array (i.e., the major and minor versions).
- join("."): This rejoins the major and minor versions with a period in between.